litmus-portal/frontend/src/views/Community/CommunityCards/index.tsx

```diff
@@ -72,16 +72,8 @@ const CommunityCards: React.FC = () => {
         role="button"
         tabIndex={0}
         className={`${classes.communityEvents} ${classes.joinCard}`}
-        onClick={() =>
-          window.open(
-            'https://www.meetup.com/Kubernetes-Chaos-Engineering-Meetup-Group'
-          )
-        }
-        onKeyDown={() =>
-          window.open(
-            'https://www.meetup.com/Kubernetes-Chaos-Engineering-Meetup-Group'
-          )
-        }
+        onClick={() => window.open('https://litmuschaos.io/community')}
```

Comment:
Please confirm this change with the Community Manager. @prithvi1307

Reply:
Sure, this is as per discussion with @ajeshbaby but I will ask @prithvi1307 and @ksatchit to verify the same.

Reply:
Verified by @prithvi1307